web-console/src/views/ingestion-view/ingestion-view.tsx

```diff
@@ -143,6 +143,8 @@ export interface IngestionViewState {
   supervisorTableActionDialogActions: BasicAction[];
   hiddenTaskColumns: LocalStorageBackedArray<string>;
   hiddenSupervisorColumns: LocalStorageBackedArray<string>;
+
+  shouldOptimizeQuery: boolean;
```

Comment:
can we make this more descriptive and instead call "useOverlord" to indicate what it's actually doing?